scripts/test_node.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
```

Comment:
Where is this script called?

Reply:
This is just a testing script to ensure the chain starts up correctly. Purely for development, not CI